src/main/resources/order-api.yml

```diff
@@ -0,0 +1,208 @@
+          "200":
+            description:  Order fields: name, date and status
+            schema:
+              type: json
+              order:
+                $ref: '#/definitions/Order'
+          "403":
+            description: Unauthorized for the current operation
+            schema:
+              $ref: '#/definitions/Error'
+          "404":
+            description: Order not found
+            schema:
+              $ref: '#/definitions/Error'
+          default:
+            description: Unexpected error
+            schema:
+              $ref: '#/definitions/Error'
+
+      /order/{order_id}/status:
+      put:
+        summary: Updates current order status
+        description: The endpoint updates current status of the order with given id.
+        The response includes the result of the operation.
+        parameters:
+        - name: id
+          in: url
+          description: Order id
+          required: true
+          type: string
+        - name: status
+          in: url
+          description: Order status
+          type: string
+        security:
+        - token: []
+        tags:
+        - order status
+        requestBody:
+          content:
+            application/text:     # Media type
+              examples: pending
+        responses:
+          "200":
+            description:  Order status
+            schema:
+              type: string
+          "403":
+            description: Unauthorized for the current operation
+            schema:
+              $ref: '#/definitions/Error'
+          "404":
+            description: Order not found
+            schema:
+              $ref: '#/definitions/Error'
+          "409":
+            description: Conflict: the order has been already taken by another business
+            schema:
+              $ref: '#/definitions/Error'
+          default:
+            description: Unexpected error
+            schema:
+              $ref: '#/definitions/Error'
+definitions:
+  Order:
+    properties:
+      order_id:
+        type: string
+        description: Unique identifier representing a specific order.
+      business_id:
+        type: string
+        description: Unique identifier representing a business, which accepted the order.
+      status:
+        type: string
+        description: Current status of an order.
+      name:
+        type: string
+        description: Client's name.
+      phone:
+        type: string
+        description: Client's phone number.
+      eMail:
+        type: string
+        description: Client's e-mail.
+      date:
+        type: date
+        description: Date, when the service is requested.
+      time:
+        type: string
+        description: Time, when the service is requested.
+      numberParticipants:
+        type: long
+        description: Number of participants on the order.
+      description:
+        type: string
+        description: Description of an order.
+    OrderList:
+      properties:
```

Comment:
It seems like it is not used anywhere. I assume you wanted to use it in get list of orders above. Lets use it. We may want to extend it later therefore using a class which wraps the array of orders - good idea.

Reply:
Thanks! It was a reference error, I corrected it in the GET method.